ros/src/tl_detector/light_classification/test_tl_classifier.py

```diff
@@ -38,6 +40,7 @@ def tearDown(self):
         TLClassifier.register_subclass("ssd-sim")(SSDSimTLClassifier)
         TLClassifier.register_subclass("ssd-real")(SSDRealTLClassifier)
         TLClassifier.register_subclass("yolo-tiny")(YOLOTinyTLClassifier)
+        TLClassifier.register_subclass("simple-cnn")(SimpleCNNTLCLassifier)
 
     def test_get_instance_of(self):
```

Comment:
You need to update an assertion. Now you have 5 known sub-types of TLClassifier instead of 4.

Reply:
Updated